### DISCUSSION TITLE
Why are coroutines on Windows pushed so far back?

### DISCUSSION THREAD

**Discussion by Wajinn** [vlang/v#19108] [General]
I would think a large percentage of V users are using or targeting the Windows OS.  According to the [roadmap](https://github.com/vlang/v/blob/master/ROADMAP.md), coroutines for Windows are pushed way out (until 1.0).  

Could anyone on team V speak on why that is so?  Thanks in advanced.

**Comment by JalonSolov** [comment]
My opinion...

1. Coroutines are hard.
2. Coroutines don't work completely on _any_ system, yet.  For example, you have to know to compile with `-gc none` to even try to use them.
3. There are no dedicated Windows developers on the V team.
4. Since there are no dedicated Windows developers on the V team, it makes 1. even harder, for Windows.
5. There are other high priority items already taking development time, such as the frontend rewrite.
